RECIPES.md

```diff
@@ -5,7 +5,7 @@ Our "Recipes" section in the sidebar highlights **step-by-step how-to guides** f
 Astro docs welcomes recipe contributions that:
 
 - Edit an [existing recipe](existing-recipes) with corrected content or [new resource links](#add-new-links), by PRing the existing file in the docs repo.
-- Add a new [community recipe](#community-recipes) by PRing a link to the ["More Recipes" page](#more-recipes-page): `src/content/docs/guides/recipes.mdx`.
+- Add a new [community recipe](#community-recipes) by PRing a link to the ["More Recipes" page](#more-recipes-page): `src/content/docs/recipes.mdx`.
```

Comment:
We also want to indicate that it is on the `en` folder, as only English pages should add more content.
```suggestion
- Add a new [community recipe](#community-recipes) by PRing a link to the ["More Recipes" page](#more-recipes-page): `src/content/docs/en/recipes.mdx`.
```